### PR TITLE
Nikhil balyan new delhi

### LINKED ISSUE #89: Recommendations From Nikhil balyan
#New Delhi, India
-[ ] create a text file
-[ ] commit changes
-[ ] push changes
-[ ] become developer
-[] test

### PR DESCRIPTION
**Pull request by nikhilbalyan**
Initial pull request closes #89.

### CODE REVIEW

**Comment by MaheshReddyV** [comment]
@nikhilbalyan :wave looks good , it would be better with more hometown information

**Comment by mahesh0309** [comment]
@nikhilbalyan Cool :+1: :wavy_dash:

**Comment by nikhilbalyan** [comment]
thanx @MaheshReddyV  @mahesh0309

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Your bleary-eyed, caffeine fueled trainers for today will be:
 ### Talk to Us! :comment:
 
 [![Join the chat at https://gitter.im/githubteacher/developers-july-AM](https://badges.gitter.im/githubteacher/developers-july-AM.svg)](https://gitter.im/githubteacher/developers-july-AM?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+- If you are with Allstate, please go here instead: https://gitter.im/githubteacher/developers-july-AM-allstate
```
Comment (mahesh0309) [inline README.md]: @nikhilbalyan Alert anything. :tongue: 
Just kidding. :smile:

**Pull request merged**